### ISSUE TITLE
inf bbox loss when using cal_giou_3d ( but the iou is right)

### ISSUE THREAD

**Issue by YuQiao0303** [lilanxiao/Rotated_IoU#49]
Hi author,
Thank you for your nice work. 
I encountered a case where I get bbox_iou of 0.7304, but bbox_loss: -inf.

After that it's:
CUDA kernel failed : invalid configuration argument.

Should I modify 
`giou_loss = 1. - iou3d + (v_c - u3d) / v_c`
as
`giou_loss = 1. - iou3d + (v_c - u3d) / (v_c + 1e-14)`?

Thank you!
